app/controllers/service_calendars_controller.rb

```diff
@@ -139,60 +106,72 @@ def move_visit_position
   end
 
   def toggle_calendar_row
```

Comment:
Added instance variables needed to rerender partials on the Service Calendar. Much of this is for eager loading.